### PR TITLE
I think this has to have a version number

### PR DESCRIPTION
**Pull request by CodeBaboon**
```
$ npm install d2l-user-tile
npm ERR! Can't install github:Brightspace/user-tile#0a3a21bd3c4e3de419b16d1e802a7c11acc45e20: Missing package version
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "polymer-cli": "^1.9.5",
     "wct-browser-legacy": "^1.0.1"
   },
-  "version": "",
+  "version": "1.0.0",
```
Comment (jstefaniuk-d2l) [inline package.json]: May I suggest a different number?

image

Comment (jstefaniuk-d2l) [inline package.json]:
```suggestion
  "version": "6.0.0",
```

Comment (CodeBaboon) [reply package.json]: I think you can literally suggest it right in the PR!

Comment (jstefaniuk-d2l) [reply package.json]: Sweet

**Pull request merged**